backbone.js

```diff
@@ -523,7 +523,9 @@
       if (this.idAttribute in attrs) {
         var prevId = this.id;
         this.id = this.get(this.idAttribute);
-        this.trigger('changeId', this, prevId, options);
+        if(this.id !== prevId) {
```

Comment:
There needs to be a space here for consistency's sake.

```suggestion
        if (this.id !== prevId) {
```